ballerina/tests/department_client.bal

```diff
@@ -41,7 +41,7 @@ client class DepartmentClient {
     }
 
     remote function readByKey(string hospitalCode, int departmentId) returns Department|error {
```

Comment:
```suggestion
    remote function readByKey(record {| string hospitalCode; int departmentId |} key) returns Department|error {
```

Reply:
since the key is a single value, isn't it more sense to have a closed record with all composite keys?

Reply:
Fixed in [8a80d26](https://github.com/ballerina-platform/module-ballerina-persist/pull/11/commits/8a80d263be98cd46f659f4c7ee8429cce0848163)